gradoop-flink/src/main/java/org/gradoop/flink/io/impl/dot/functions/AbstractDotFileFormat.java

```diff
@@ -15,58 +15,35 @@
  */
 package org.gradoop.flink.io.impl.dot.functions;
 
-import  static org.apache.commons.lang3.StringEscapeUtils.escapeHtml4;
-
-import java.util.Iterator;
-
-import org.apache.commons.lang3.StringUtils;
 import org.apache.flink.api.java.io.TextOutputFormat;
 import org.gradoop.common.model.api.entities.EPGMElement;
 import org.gradoop.common.model.impl.id.GradoopId;
 import org.gradoop.common.model.impl.pojo.Edge;
 import org.gradoop.common.model.impl.pojo.GraphHead;
-import org.gradoop.common.model.impl.pojo.Vertex;
-import org.gradoop.common.model.impl.properties.Properties;
-import org.gradoop.common.model.impl.properties.Property;
 import org.gradoop.flink.model.impl.layouts.transactional.tuples.GraphTransaction;
 
 /**
- * Converts a GraphTransaction to the following .dot format:
- * <p>
- *   digraph 0
- *   {
- *   gradoopId1 [label="person",name="Bob",age="20",...];
- *   gradoopId2 [label="person",name="Alice",age="20",...];
- *   gradoopID3;
- *   gradoopID4;
- *   gradoopId1->gradoopId2 [label="knows",since="2003",...];
- *   gradoopId2->gradoopId1 [label="knows",since="2003",...];
- *   gradoopId3->gradoopId4;
- *   }
- * </p>
+ * Base class for implementing a dot formatting.
  */
-public class DOTFileFormat implements TextOutputFormat.TextFormatter<GraphTransaction> {
+public abstract class AbstractDotFileFormat
+  implements TextOutputFormat.TextFormatter<GraphTransaction> {
+
   /**
-   * id that needs to be update upon changes to this class' structure.
+   * .DOT vertex identifier prefix
    */
-  private static final long serialVersionUID = 1L;
+  static final String VERTEX_ID_PREFIX = "v";
+
   /**
-   * .DOT vertex identifier prefix
+   * id that needs to be update upon changes to this class' structure.
    */
-  private static final String VERTEX_ID_PREFIX = "v";
+  private static final long serialVersionUID = 1L;
```

Comment:
Since you are changing this class, you could change the serial ID, but this should not matter.